invenio_rdm_records/resources/serializers/dublincore/__init__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Dublin Core Serializers for Invenio RDM Records."""
+
+from dcxml import simpledc
+from flask_resources.serializers import MarshmallowJSONSerializer, \
+    SerializerMixin
+
+from .schema import DublinCore
+
+
+class DublinCoreJSONSerializer(MarshmallowJSONSerializer):
+    """Marshmallow based Dublin Core serializer for records."""
+
+    def __init__(self, **options):
+        """Constructor."""
+        super().__init__(schema_cls=DublinCore, **options)
+
+    def dump_one(self, obj):
+        """Dump the object with extra information."""
+        json = self._schema_cls().dump(obj)
+        return json
+
+
+class DublinCoreXMLSerializer(SerializerMixin):
```

Comment:
you might need to implement `serialize_object_list` too (see mixin)... otherwise it will fail for a list of records.

Reply:
implemented now. there is also a test case for it